### PR TITLE
Mfw 3949 : Add VRRP track objects into mfw_schema

### PR DESCRIPTION
**Pull request by trilok-venkata**
Changes proposed added at : https://awakesecurity.atlassian.net/browse/MFW-3949?focusedCommentId=231755
Added a new schema called as track which would be used as below 

{
    "track": [
        {
            "name" : "LAN0",
            "description" : "Tracking Object for LAN0",
            "interfaceId" : "2"
        },
        {
            "name" : "WAN0",
            "description" : "Tracking Object for WAN0",
            "interfaceId" : "3"
        }
    ]
}

The VRRP tracking object will be created as below :
interface Ethernet1/1
   vrrp 10 ipv4 172.30.174.55
   vrrp 10 priority-level 150
   vrrp 10 tracked-object LAN1 decrement 10

Added a new vrrptrack to interface_settings.
Validation results:

================================================
python3 validate_test_schema.py

.
----------------------------------------------------------------------
Ran 1 test in 0.001s

OK
$

=====================================================

### CODE REVIEW

File: v1/track/validate_test_schema.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+	def test_track_configs(self):
+		"""
+		validates track configs
+		"""
+		track_objects = self.json_data["track"]
+
+		for track_object in track_objects:
+			self.assertTrue(track_object["name"] in ["LAN0", "WAN0"], "Failed due to an invalid track object name")
+			self.assertTrue(track_object["description"], "Failed due to an empty description")
```
Comment (AlistairJohnParker) [inline v1/track/validate_test_schema.py:64]: description is not required

File: v1/track/validate_test_schema.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+			raise unittest.SkipTest("ERROR: Validation of schema failed. Skipping all tests and printing.")
+
+	def test_track_configs(self):
+		"""
+		validates track configs
+		"""
+		track_objects = self.json_data["track"]
+
+		for track_object in track_objects:
+			self.assertTrue(track_object["name"] in ["LAN0", "WAN0"], "Failed due to an invalid track object name")
```
Comment (AlistairJohnParker) [inline v1/track/validate_test_schema.py:63]: Ok...only allowing LAN0 and WAN0 ? Is that correct ?

**Comment by AlistairJohnParker** [comment]
LGTM  - meets the requirements AFAICT

File: v1/track/validate_test_schema.py (hunk 1 above)
Comment (trilok-venkata) [reply v1/track/validate_test_schema.py]: Addressed

File: v1/track/validate_test_schema.py (hunk 2 above)
Comment (trilok-venkata) [reply v1/track/validate_test_schema.py]: Actually intention was to ensure name field was passed.
I have addressed the validation by ensuring the name field is not empty  on the latest diff.

**Pull request merged**